packages/ag-charts-types/src/series/cartesian/rangeAreaOptions.ts

```diff
@@ -27,13 +27,20 @@ export interface AgRangeAreaSeriesStyle extends StrokeOptions, FillOptions, Line
     marker?: AgSeriesMarkerStyle;
 }
 
+export type AgRangeAreaSeriesItemType = 'high' | 'low';
+
 export interface AgRangeAreaSeriesTooltipRendererParams<TDatum = DatumDefault, TContext = ContextDefault>
     extends Omit<AgCartesianSeriesTooltipRendererParams<TDatum, TContext>, 'xKey' | 'xName' | 'yKey' | 'yName'>,
         AgRangeAreaSeriesOptionsKeys<TDatum>,
         AgRangeAreaSeriesOptionsNames,
         Omit<AgSeriesMarkerStyle, 'shape'> {
-    /** Hovered marker */
-    itemId: 'up' | 'down';
```

Comment:
Reviewed offline - practically we received `high` or `low` as `itemId` values going back to `9.0.0`, so this is a fixing breaking change.